client_test.go

```diff
@@ -224,6 +224,40 @@ func TestAddRecordToZone(t *testing.T) {
 	require.NotNil(t, rs)
 }
 
+func TestSelectFilteredRRSetsFromZone(t *testing.T) {
```

Comment:
Should we test `WithoutResourceRecordSet` as well?

Reply:
Yes, we should. Will add tomorrow. 🥱